tests/deployment/test_deployment.py

```diff
@@ -0,0 +1,10 @@
+import logging
+
+from deployment.deployment import Deployment
+
+log = logging.getLogger(__name__)
+
+
+def test_deployment():
```

Comment:
Should not be in test file starting with test_ prefix. Rather to have package called entry_points, and there you can have separate module for deployment entry point, and another for destroy later on.